File: sdk/async-scala/src/main/scala/io/swagger/client/model/AddressAssignment.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class AddressAssignment (
  AddressID: Option[String],
UserID: Option[String],
UserGroupID: Option[String],
IsShipping: Option[Boolean],
IsBilling: Option[Boolean]
)

File: sdk/javascript-closure-angular/API/Client/AddressAssignment.js
goog.provide('API.Client.AddressAssignment');

/**
 * @record
 */
API.Client.AddressAssignment = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.AddressAssignment.prototype.addressID;

/**
 * @type {!string}
 * @export
 */
API.Client.AddressAssignment.prototype.userID;

/**
 * @type {!string}
 * @export
 */
API.Client.AddressAssignment.prototype.userGroupID;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AddressAssignment.prototype.isShipping;

/**
 * @type {!boolean}
 * @export
 */
API.Client.AddressAssignment.prototype.isBilling;

